libsyclinterface/helper/source/dpctl_error_handlers.cpp

```diff
@@ -40,6 +41,29 @@ void DPCTL_AsyncErrorHandler::operator()(
     }
 }
 
+class GLogHelper
+{
+public:
+    GLogHelper()
+    {
+        google::InitGoogleLogging("");
+        google::EnableLogCleaner(0);
+        google::InstallFailureSignalHandler();
+        FLAGS_colorlogtostderr = true;
+        FLAGS_stderrthreshold = google::FATAL;
+        if (FLAGS_log_dir.empty()) {
+            FLAGS_log_dir = "/tmp";
+        }
+    }
+
+    ~GLogHelper()
+    {
+        google::ShutdownGoogleLogging();
+    }
+};
+
+static GLogHelper gloginit;
```

Comment:
If you declare it inside the anonymous namespace, the effect is the same. `static` keyword is a legacy C feature and modern C++ should use the anonymous namespace.